pennylane/decomposition/__init__.py

```diff
@@ -101,11 +130,107 @@ def my_cnot(wires):
 Integration with the Decompose Transform
```

Comment:
I suggest putting this above the "Graph based decompositio solver" section!

Reply:
Fixed [e7b3a4d](https://github.com/PennyLaneAI/pennylane/pull/6966/commits/e7b3a4d3a1469453fbe7623c58bf7f3120258ffd)